ddl/column.go

```diff
@@ -347,7 +347,8 @@ func (d *ddl) backfillColumnInTxn(t table.Table, colMeta *columnMeta, handles []
 			return 0, errors.Trace(err)
 		}
 
-		rowColumns, err := tablecodec.DecodeRow(rowVal, colMeta.oldColMap)
+		// TODO: Should use session's TimeZone instead of UTC.
```

Comment:
I think the best way is to convert the default value to UTC timezone in ddl API.
So we can just use UTC to decode row in DDL worker.